webapp/sources/rudder/rudder-web/src/main/resources/rudder-system-global-parameter.conf

```diff
@@ -0,0 +1,10 @@
+[{
```

Comment:
shouldn't it be a .json file instead?

Reply:
I prefer to have a `.conf` or whatever to not be bound to json if we evolve format to hocon. And since it's internal, I think it doesn't matter.